packages/zowe-explorer-api/src/profiles/ProfilesCache.ts

```diff
@@ -150,6 +150,7 @@ export class ProfilesCache {
             return;
         }
         const allTypes = this.getAllProfileTypes(apiRegister?.registeredApiTypes() ?? []);
+        allTypes.push("ssh");
```

Comment:
This was added for extenders calling loadNamedProfile() or other profile fetching APIs for a SSH profile. possible regression as it previously worked.

Reply:
Sounds like a good idea to resolve the regression for now, but curious if we should discuss this a bit more 😋 

I guess I'm trying to understand how/when the regression started, since I don't recall us having to add the ssh profile as part of the loaded types.

Personal guess: Maybe because we moved to individual SDKs?

Reply:
Also curious why we need to hardcode "ssh" here, since z/OSMF and TSO profiles seem to work without being manually added here.

Reply:
zosmf is added to the registered profile list when registered during activation, left comment about double checking tso if it's a regression and possibly updating to register the other types similar to how cics profiles register.